contrib/fixup-libgfortran.sh

```diff
@@ -41,8 +42,22 @@ find_shlib()
     fi
 }
 
+find_shlib_dir()
+{
+    # Usually, on platforms like OSX we get full paths when linking.  However,
+    # if we are inspecting, say, BinaryBuilder-built OpenBLAS libraries, we will
+    # only get something like `@rpath/libgfortran.5.dylib` when inspecting the
+    # libraries.  We can, as a last resort, ask `$FC` directly what the full
+    # filepath for this library is, but only if we don't have a direct path to it:
+    if [[ $(dirname "$1") == "@rpath" ]]; then
```

Comment:
This file uses `/bin/sh`, but some of the changes in this PR are using Bash extensions, e.g. `[[`.

Reply:
nice catch; fixed those ones